.github/workflows/publish_develop.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish Develop App
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  publish:
+    name: 🚀 Publish App to Develop Channel
+    runs-on: ubuntu-latest
+    steps:
+      - name: 🏗 Setup repo
+        uses: actions/checkout@v3
+
+      - name: 🏗 Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
```

Comment:
I tested the deployment with node 16 and it works fine. I had not seen that we have node 14 on the engines. Node 14 is currently in maintenance mode. Why are we using it instead of v16, the Active LTS version?

Reply:
I guess we hadn't considered whether the node we use is LTS. We went with what the majority used ( at the initial stages of the project )
We set node engine as 14 to regulate lockfileversion changing every time sb npm installs.
https://github.com/droidconKE/droidconKE2022ReactNative/pull/1#pullrequestreview-1094526098
However lockfileversion still changed even after setting engine. 
We wanted to create a separate issue to research on thism

Reply:
I changed it to v14.

Reply:
Would be nice to consider upgrading to node 16 in the future.

Reply:
Totally. I'm tempted to push for this right now ( since we are at initial stages of the project )